js/packages/phoenix-client/examples/create_prompt_openai.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-console */
+import { createPrompt, promptVersion } from "../src/prompts";
+import { toSDK } from "../src/prompts";
+import OpenAI from "openai";
+
+const openai = new OpenAI({
```

Comment:
The other examples define the client as well for easy access to base url / API key config. Not sure if you think it's worth doing that in every example of not

Reply:
done